### PR TITLE
removes the builtin function syntax

### PR DESCRIPTION
**Pull request by christophkloeffel**
#51

### CODE REVIEW

File: tests-system/builtin-2/legacy.rsl
Comment (florianschanda) [inline tests-system/builtin-2/legacy.rsl:1]: We should not delete tests, just update the name and/or output.

**Review by florianschanda — CHANGES_REQUESTED** [review]
Nice work :)

Removing this means we need to change the major version. It also means we should kill check files at that point and possible make `-I .` the default as well. To be discussed...

Some comments:

lexer.py:

* You need to also remove the `:` option for the identifier loop
  in the lexer, as `:` can no longer be part of a valid name

CHANGELOG.md:

* You need an entry here as well

**Comment by phiwuu** [comment]
This PR is open for a while now. For me it is okay to step to the next major version now. Shall we merge this PR?

**Comment by florianschanda** [comment]
We can, but it'd be good to bundle it with some other incompatible changes. Do you have a wishlist?

**Pull request merged**